src/server/getters/get-markets.ts

```diff
@@ -39,6 +42,12 @@ export async function getMarkets(db: Knex, augur: {}, params: t.TypeOf<typeof Ge
     });
   }
 
-  const marketsRows = await queryModifier<MarketsContractAddressRow>(db, query, "volume", "desc", params);
-  return marketsRows.map((marketsRow: MarketsContractAddressRow): Address => marketsRow.marketId);
+  let marketsRows = await queryModifier<MarketsContractAddressAndFeesRow<BigNumber>>(db, query, "volume", "desc", params);
+
+  if (params.maxFee != null && params.maxFee !== undefined) {
+    const maxFee: number = params.maxFee;
+    marketsRows = marketsRows.filter((row: MarketsContractAddressAndFeesRow<BigNumber>) => row.reportingFeeRate.plus(row.marketCreatorFeeRate).lte(maxFee));
```

Comment:
I'm pretty sure I know the answer but can this be done on the Knex side? Perhaps using one of those old-timey SQL solutions?

Reply:
Something like:

```
if (params.maxFee != null && params.maxFee !== undefined) query.whereRaw("(CAST(markets.reportingFeeRate as numeric) + CAST(markets.marketCreatorFeeRate as numeric)) >= ?", [params.maxFee]);
```

Reply:
I'll test it out. We don't do this anywhere else so I had assumed there was some reason not